magefile.go

```diff
@@ -0,0 +1,496 @@
+//go:build mage
+// +build mage
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"go.uber.org/multierr"
+)
+
+var _depBin = filepath.Join(_dependencyDir, "bin")
+
+var _dependencyDir = func() string {
+	if dir, ok := os.LookupEnv("DEPENDENCY_DIR"); ok {
+		return dir
+	}
+
+	return filepath.Join(_projectRoot, ".cache", "dependencies")
+}()
+
+var _projectRoot = func() string {
+	if root, ok := os.LookupEnv("PROJECT_ROOT"); ok {
+		return root
+	}
+
+	root, err := sh.Output("git", "rev-parse", "--show-toplevel")
+	if err != nil {
+		panic("failed to get working directory")
+	}
+
+	return root
+}()
+
+var _tag = func() string {
+	tag, err := sh.Output("git", "rev-parse", "--short", "HEAD")
+	if err != nil {
+		panic("failed to get current tag")
+	}
+
+	return tag
+}()
+
+var Aliases = map[string]interface{}{
+	"check":     All.Check,
+	"clean":     All.Clean,
+	"run-hooks": Hooks.Run,
+	"test":      All.Test,
+}
+
+type All mg.Namespace
+
+func (All) Check(ctx context.Context) {
```

Comment:
Go imports got lost during the transition. Was this intended?

Reply:
`goimports` and `gofmt` are included with `golangci-lint` so I just enabled them:
`"-E", "unused,gofmt,goimports,gosimple,staticcheck",`